modin/experimental/xgboost/xgboost_ray.py

```diff
@@ -382,6 +559,38 @@ def _train(
     evals=(),
     **kwargs,
 ):
+    """Run distributed training of XGBoost model on Ray backend.
+
+    During work it evenly distributes `dtrain` between workers according
+    to IP addresses partitions (in case of not evenly distribution of `dtrain`
+    is possible by IPs, part of partitions will be re-distributed between nodes),
```

Comment:
```suggestion
    to IP addresses partitions (in case of not even distribution of `dtrain`
    by nodes, part of partitions will be re-distributed between nodes),
```

Reply:
fixed